influxdb3/client.go

```diff
@@ -171,7 +171,7 @@ func (c *Client) makeAPICall(ctx context.Context, params httpParams) (*http.Resp
 		}
 	}
 	req.Header.Set("User-Agent", userAgent)
-	if c.authorization != "" {
+	if c.authorization != "" && req.Header.Get("Authorization") == "" {
```

Comment:
Please add test to this change into `client_test.go`.

Reply:
Addressed in [d5b0fb9](https://github.com/InfluxCommunity/influxdb3-go/pull/87/commits/d5b0fb9cb5ba790990e6e383b255ddae1b4c4f42)